### PR TITLE
[Misc] Update version

### PR DESCRIPTION
**Pull request by Accelerator1996**

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-17.0.12.0.13+7-GA
\ No newline at end of file
+17.0.13.0.14+9-GA
```
Comment (Accelerator1996) [inline version.txt:1]: diff --git a/version.txt b/version.txt
index 2278dd14794..2f91f298871 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-17.0.13.0.14+9-GA
\ No newline at end of file
+17.0.12.0.13+7-GA
\ No newline at end of file

Comment (Accelerator1996) [inline version.txt:1]:
```
diff --git a/version.txt b/version.txt
index 2f91f298871..2278dd14794 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-17.0.12.0.13+7-GA
\ No newline at end of file
+17.0.13.0.14+9-GA
\ No newline at end of file
```

**Review by kuaiwei — APPROVED** [review]
LGTM

**Review by sendaoYan — APPROVED** [review]
/LGTM

**Pull request merged**